### DISCUSSION TITLE
How to hide performance monitor on iPad?

### DISCUSSION THREAD

**Discussion by degtiarev** [maplibre/maplibre-native#2785] [Q&A] [answered]
That might be a stupid question, but i cannot find how to hide performance monitor that appears on iPad. It appeared after update to version 6.x. 

<img width="1180" alt="image" src="https://github.com/user-attachments/assets/4ab0691b-303d-48ff-b2b2-d3b416aefb0b">

**Comment by louwers** [comment, marked as answer]
https://developer.apple.com/documentation/xcode/monitoring-your-metal-apps-graphics-performance

Affects all Metal apps.

**Comment by degtiarev** [comment]
Thanks! I thought it's a part of MapLibre. Spent couple hours trying to find a way to switch it off.

**Reply by louwers** [reply]
Oh no! Feel free to join our Slack channel with MapLibre devs and users, link is in the README.
